gum/arch-x86/gumx86writer.c

```diff
@@ -832,11 +807,14 @@ gum_x86_writer_put_jcc_short (GumX86Writer * self,
 
 void
 gum_x86_writer_put_jcc_near (GumX86Writer * self,
-                             guint8 opcode,
+                             x86_insn instruction_id,
                              gconstpointer target,
                              GumBranchHint hint)
 {
   gssize distance;
+  guint8 opcode;
+
+  opcode = gum_get_jcc_opcode (instruction_id);
```

Comment:
I guess we could drop the variable as doing the call inline would be equally clear?